src/test/java/org/icgc/argo/program_service/services/EgoServiceIT.java

```diff
@@ -124,13 +138,53 @@ void joinAndLeaveProgram() {
 
     val groupId = egoService.getGroup("PROGRAM-TestShortName-ADMIN").get().getId();
 
-    val user = egoService.getObject(String.format("%s/groups/%s/users?query=%s", appProperties.getEgoUrl(), groupId, "d8660091@gmail.com"), new ParameterizedTypeReference<EgoService.EgoCollection<EgoService.User>>() {});
+    val user = egoService.getObject(String.format("%s/groups/%s/users?query=%s", appProperties.getEgoUrl(), groupId, "d8660091@gmail.com"), new ParameterizedTypeReference<EgoService.EgoCollection<EgoService.EgoUser>>() {});
     assertThat(user.isPresent()).isTrue();
 
     egoService.leaveProgram("d8660091@gmail.com", programEntity.getId());
+    assertThat(egoService.getObject(String.format("%s/groups/%s/users?query=%s", appProperties.getEgoUrl(), groupId, "d8660091@gmail.com"), new ParameterizedTypeReference<EgoService.EgoCollection<EgoService.EgoUser>>() {}).isPresent()).isFalse();
+  }
 
+  @Test
+  void listUser(){
+    List<String> expectedUsers = new ArrayList();
+    expectedUsers.add(ADMIN_USER_EMAIL);
+    expectedUsers.add(COLLABORATOR_USER_EMAIL);
+
+    val adminGroupId = egoService.getGroup("PROGRAM-TestShortName-ADMIN").get().getId();
+    val collaboratorGroupId = egoService.getGroup("PROGRAM-TestShortName-COLLABORATOR").get().getId();
+
+    // 1. add an ADMIN user to TestProgram
+    val adminJoin = egoService.joinProgram(ADMIN_USER_EMAIL, programEntity, UserRole.ADMIN);
+    assertThat(adminJoin).isTrue();
```

Comment:
assertThat(adminJoin).as("Can add ADMIN user to TestProgram").isTrue()
When fails 
 [Can add ADMIN user to TestProgram] expected:<true> but was:<false>